assets/model_monitoring/components/data_drift/data_drift_signal_monitor/spec.yaml

```diff
@@ -190,7 +190,7 @@ jobs:
       type: aml_token
   output_signal_metrics:
     type: spark
-    component: azureml://registries/azureml/components/model_monitor_output_metrics/versions/0.3.7
```

Comment:
you also need to change the version of this signal at the top.

Reply:
done